src/ERC20DelegatesUpgradeable.sol

```diff
@@ -86,27 +92,25 @@ abstract contract ERC20DelegatesUpgradeable is Initializable, ERC20Upgradeable,
 
     /* INTERNAL */
 
-    /// @dev Delegates all of `account`'s voting units to `delegatee`.
-    /// @dev Emits events {IDelegates-DelegateChanged} and {IDelegates-DelegateVotesChanged}.
-    function _delegate(address account, address delegatee) internal {
+    /// @dev Delegates the balance of the `delegator` to `delegatee`.
+    function _delegate(address delegator, address delegatee) internal {
         ERC20DelegatesStorage storage $ = _getERC20DelegatesStorage();
-        address oldDelegate = delegates(account);
-        $._delegatee[account] = delegatee;
+        address oldDelegate = delegates(delegator);
+        $._delegatee[delegator] = delegatee;
 
-        emit DelegateChanged(account, oldDelegate, delegatee);
-        _moveDelegateVotes(oldDelegate, delegatee, _getVotingUnits(account));
+        emit DelegateChanged(delegator, oldDelegate, delegatee);
+        _moveDelegateVotes(oldDelegate, delegatee, _getVotingUnits(delegator));
     }
 
-    /// @dev Must return the voting units held by an account.
-    function _getVotingUnits(address account) internal view returns (uint256) {
-        return balanceOf(account);
+    /// @dev Returns the voting units held by a delegator.
+    function _getVotingUnits(address delegator) internal view returns (uint256) {
+        return balanceOf(delegator);
     }
 
     /// @dev Moves voting power when tokens are transferred.
     /// @dev Emits a {IDelegates-DelegateVotesChanged} event.
     function _update(address from, address to, uint256 value) internal virtual override {
         super._update(from, to, value);
-        // No check of supply cap here like in OZ implementation as MORPHO has a 1B total supply cap.
```

Comment:
A comment somewhere on the topic would be good, maybe the README?

Reply:
I think we should keep this comment

Reply:
OZ checks the totalSupply on mint, so why should we speak about this?

```solidity
        if (from == address(0)) {
            // Overflow check required: The rest of the code assumes that totalSupply never overflows
            _totalSupply += value;
```